ext/civiimport/templates/CRM/Import/Monitor.tpl

```diff
@@ -0,0 +1,15 @@
+<div id='crm-import-progress'></div>
+{literal}
+  <script>
+    var target = '#crm-import-progress';
+    var url = CRM.vars.civiimport.url;
+    // Load the snippet into the container
+    CRM.loadPage(url, {
+      target: target,
+      block: false
+    })
+    window.setInterval(function() {
+      CRM.$(target).crmSnippet('refresh');
+    }, 1000);
```

Comment:
JS code needs to be enclosed otherwise variables are in the global scope.

```suggestion
   CRM.$(function() {
     var target = '#crm-import-progress';
     var url = CRM.vars.civiimport.url;
     // Load the snippet into the container
     CRM.loadPage(url, {
       target: target,
       block: false
     })
     window.setInterval(function() {
       $(target).crmSnippet('refresh');
     }, 1000);
   });
```

Reply:
thanks @colemanw - I fixed this & retested it & it worked :-)